src/components/transport_manager/test/include/transport_manager/transport_adapter/mock_transport_adapter_controller.h

```diff
@@ -44,7 +44,7 @@ using namespace ::transport_manager::transport_adapter;
 
 class MockTransportAdapterController : public TransportAdapterController {
  public:
-  MOCK_METHOD1(AddDevice, DeviceSptr(DeviceSptr device));
+  MOCK_METHOD1(AddDevice, DeviceSptr(std::string device));
```

Comment:
This change probably shouldn't compile, this does not match the method signature in the TransportAdapterController. It's possible that it just isn't used anywhere, but this change should be reverted anyway.